samples/meetings-content-bubble/csharp/assets/sample.json

```diff
@@ -3,10 +3,10 @@
     "name": "officedev-microsoft-teams-samples-meetings-content-bubble-csharp",
     "source": "officeDev",
     "title": "Content Bubble Meeting app",
-    "shortDescription": "Illustrates how to implement the \"content bubble\" in-meeting experience",
+    "shortDescription": "Illustrates how to implement the \"content bubble\" and \"targeted notifications\" in-meeting experience",
     "url": "https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/meetings-content-bubble/csharp",
     "longDescription": [
-      "Illustrates how to implement the \"content bubble\" in-meeting experience"
+      "Illustrates how to implement the \"targeted in-meeting notifications\" in-meeting experience"
```

Comment:
Minor: This seems a bit weird as we are using in-meeting twice. Maybe we can update it to be
"Illustrate how to implement the \"targeted in-meeting notifications\" experience" or 
"Illustrate how to implemenet the \"targeted meeting notifications\" in meetings".

Reply:
Yes updated it